Src/Couchbase.Linq/Execution/IBucketQueryExecutor.cs

```diff
@@ -28,7 +28,7 @@ internal interface IBucketQueryExecutor : IQueryExecutor
         /// <summary>
         /// Specifies if the query results should be streamed, reducing memory utilzation for large result sets.
         /// </summary>
-        /// <remarks>The default is true.</remarks>
+        /// <remarks>The default is false.</remarks>
         bool UseStreaming { get; set; }
```

Comment:
Potentially breaking change; imagine a somebody has a large query that previously worked and now they get an OOM.